middlewares/middlewares.go

```diff
@@ -73,7 +74,11 @@ func getRemoteAddr(r *http.Request) string {
 	if len(address) > 0 {
 		return address
 	}
+	if splitIndex := strings.Index(r.RemoteAddr, ":"); splitIndex > 0 {
```

Comment:
You'll probably need to use `strings.LastIndex()` to account for ipv6 `[::]:1234`